src/sudoers/tokens.rs

```diff
@@ -185,7 +185,13 @@ impl Token for Command {
             // if no arguments are mentioned, anything is allowed
             None
         } else {
-            if args.last().map(|x| -> &str { x }) == Some("\"\"") {
```

Comment:
This PR also replaces this really cute implicit usage of the Deref coercion with an explicit use of `.is_some_and()` (which wasn't in stable Rust when this code was written). I feel a bit sad about this change.